cmake/modules/platform/toolcfg/gnu.cmake

```diff
@@ -128,12 +128,22 @@ function(_omr_toolchain_separate_debug_symbols tgt)
 		omr_get_target_output_genex(${tgt} output_name)
 		set(dbg_file "${output_name}${OMR_DEBUG_INFO_OUTPUT_EXTENSION}")
 		if(OMR_OS_AIX AND CMAKE_C_COMPILER_IS_OPENXL)
-			add_custom_command(
-				TARGET "${tgt}"
-				POST_BUILD
-				COMMAND "${CMAKE_COMMAND}" -E copy ${exe_file} ${dbg_file}
-				COMMAND "${CMAKE_STRIP}" -X32_64 ${exe_file}
-			)
+			# When building with OpenXL on AIX, get rid of strip for libjvmtitest.so until
+			# https://github.com/eclipse-openj9/openj9/issues/21528 is resolved
```

Comment:
Please finish each sentence with a period.

It doesn't feel right to make an exception here for `jvmtitest` which would otherwise be unknown to OMR.

Reply:
Is there somewhere in the OpenJ9 codebase I can put it then? I did it here initially since this is where the `strip` command is added to the build steps, but if there's a way to implement this workaround that puts this change in a more intuitive spot, then I can definitely do that instead.

Reply:
There could be a DONT_STRIP_MODULE variable that is empty in OMR and that OPENJ9 sets.